### PR TITLE
package.json dependencies update

### PR DESCRIPTION
**Pull request by vyakymenko**
* dependencies update.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,11 +97,11 @@
     "stream-series": "^0.1.1",
     "stylelint": "^7.0.2",
     "stylelint-config-standard": "^11.0.0",
-    "systemjs-builder": "0.15.14",
+    "systemjs-builder": "0.15.23",
```
Comment (mgechev) [inline package.json]: This version is locked intentionally.

File: package.json [hunk 2]
```diff
@@ -122,8 +122,8 @@
     "@angular/router": "3.0.0-beta.2",
     "es6-module-loader": "^0.17.8",
     "core-js": "^2.4.0",
-    "rxjs": "5.0.0-beta.6",
-    "systemjs": "0.19.27",
+    "rxjs": "5.0.0-beta.10",
```
Comment (mgechev) [inline package.json]: This update causes the build failure.

Comment (vyakymenko) [inline package.json]: @mgechev , you right, i have reverted this change.

File: package.json (hunk 1 above)
Comment (vyakymenko) [inline package.json]: @mgechev , my false, reverted.

**Comment by vyakymenko** [comment]
@mgechev , re-test it, now everything ok.

**Comment by AloeDream** [comment]
One of new packages causes this to fail. Had exact same problem with prod build when manually updated everything except rxjs to latest versions. Spent hours then rolled package.json back and prod build started performing well.

**Pull request merged**